Jenkinsfile

```diff
@@ -69,6 +73,35 @@ for(i = 0; i < buildTypes.size(); i++) {
 builds.failFast = failFast
 parallel builds
 
+stage("Acceptance Tests") {
+    node('docker && highmem') {
+        timestamps {
+            dir("ath") {
+                git url: "https://github.com/jenkinsci/acceptance-test-harness.git"
```

Comment:
Pulling from the `master` branch will make core PRs fail pretty much at random. This is unacceptable IMO.

Reply:
So, would hardcoding a release tag we would advance every once in a while work better for you?

Reply:
Yes, a release tag would be fine. (Even a commit hash from today’s head of `master` would work, though it is less obvious in that case when updating the hash that you are going forward.)